samples/reactjs-todo/client/components/journey/form.js

```diff
@@ -53,15 +52,15 @@ export default function Form() {
         methods.setEmail(user.email);
         methods.setAuthentication(true);
 
-        history.push('/');
+        navigate('/todos');
       } catch (err) {
         console.error(`Error: token or userinfo request; ${err}`);
       }
     }
     if (isAuthenticated) {
       oauthFlow();
     }
-  }, [isAuthenticated]);
+  }, [methods, navigate, isAuthenticated]);
```

Comment:
We shouldn't need to add `methods` or `navigate` to the array as they are stable references and won't change. Did ESLint bark at you for not having them? If so, that's why I placed the ESLint `disable-next-line` comments turning that off after evaluating the dependency array.